### PR TITLE
Repackage as io.trino

### PR DESCRIPTION
**Pull request by wendigo**
cc @electrum

### CODE REVIEW

File: trino-jdbc/src/main/resources/META-INF/services/java.sql.Driver [hunk 1]
```diff
@@ -1 +1 @@
-io.prestosql.jdbc.PrestoDriver
+io.trino.jdbc.PrestoDriver
```
Comment (wendigo) [inline trino-jdbc/src/main/resources/META-INF/services/java.sql.Driver]: We could add a class extending `io.trino.jdbc.PrestoDriver` to ease with migration.

File: migrate.sh [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+git commit -m "Repackage as io.trino (step 2 of 4)" -m "Update package statements"
+
+echo "Migrating imports in Java files"
+
+find . -type f -name '*.java' | while read file; do
+    echo "Fixing imports in ${file}"
+
+    sed -i '
+        s/io\.prestosql/io\.trino/g
+        s/io\/prestosql/io\/trino/g
```
Comment (findepi) [inline migrate.sh:39]: This is in string constants, resource locations?

**Review by findepi — COMMENTED** [review]
"Add migration script to io.trino" LGTM

File: trino-hive/src/main/java/io/prestosql/plugin/hive/s3/PrestoS3FileSystem.java [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.plugin.hive.s3;
+
+public class PrestoS3FileSystem
```
Comment (findepi) [inline trino-hive/src/main/java/io/prestosql/plugin/hive/s3/PrestoS3FileSystem.java]: Deprecate & add comment why it's needed?

**Comment by electrum** [comment]
Thanks for your work on this. Superseded by #6484

**Pull request closed without merging**